app/models/patron.rb

```diff
@@ -151,6 +151,16 @@ def to_partial_path
 
   private
 
+  def borrow_direct_requests
+    return [] if proxy_borrower? # Proxies can't submit borrow direct requests, so don't check.
```

Comment:
Just confirming that fee borrowers can do BD?

Reply:
I would guess probably not.  @saseestone, can you confirm?

Reply:
Extracted to #326